### ISSUE TITLE
Support of collection.parse method in fetching methods

### ISSUE THREAD

**Issue by ulitiy** [alekseykulikov/backbone-offline#12]
parse method should be called each time collection is fetched from the server

**Comment by mmrose** [comment]
I agree, because not calling collection.parse breaks backbone-tastypie (https://github.com/PaulUithol/backbone-tastypie), which relies on parsing the collection.
